### PR TITLE
IDVA5-272-confirmation screen

### PR DESCRIPTION
**Pull request by abhiahirv1**

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by ttingle-ch — APPROVED** [review]

File: src/services/formatService.ts [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    public static formatHowIdentityDocsChecked (option: string | undefined, lang: string): string {
+        if (!option) {
+            return "";
+        }
+        let result: string;
+
+        switch (option) {
+        case "OPTION1":
+            if (lang === "en") {
+                result = "Option 1 - The identity documents were checked using identity document validation technology (IDVT).";
```
Comment (sampankumar) [inline src/services/formatService.ts:148]: Why is it hard coded value here? Why not picking it up from json file?

File: src/services/formatService.ts [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+            } else {
+                result = "Option 1 - The identity documents were checked using identity document validation technology (IDVT). Welsh";
+            }
+            break;
+        case "OPTION2":
+            if (lang === "en") {
+                result = "Option 2 - The identity documents were checked by a person.";
```
Comment (sampankumar) [inline src/services/formatService.ts:155]: same as above

File: src/services/formatService.ts (hunk 1 above)
Comment (abhiahirv1) [reply src/services/formatService.ts:148]: We can not pickup json within service also we have a different text which is on there in json.

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by sampankumar — APPROVED** [review]

**Pull request merged**